### PR TITLE
Improved installation doc.

### PR DESCRIPTION
**Pull request by endalk200**
I  have changed some terms that makes the description in the documentation vague. and fix some typo errors to make the documentation
easy to understand for all language backgrounds.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -116,8 +116,8 @@ Debian / Ubuntu
 WeasyPrint is `packaged for Debian 11 or newer
 <https://packages.debian.org/search?searchon=names&keywords=weasyprint>`_.
 
-You can install it with pip on Debian 10 Buster or newer, or on Ubuntu 18.04
-Bionic Beaver or newer, after installing the following packages:
+You can install it with pip on Debian 10 Buster or latest versions, or on Ubuntu 18.04
+Bionic Beaver or lastest versions, after installing the following packages:
```
Comment (Tontyna) [inline docs/install.rst]: `latest versions` isn't correct. It's indeed `or newer`

File: docs/install.rst [hunk 2]
```diff
@@ -245,8 +245,8 @@ packages. They are part of `GTK+ <https://en.wikipedia.org/wiki/GTK+>`_
 The following installation instructions for the GTK+ libraries don't work on
 Windows XP. That means: Windows Vista or later is required.
 
-Of course you can decide to install ancient WeasyPrint versions with an
-erstwhile Python, combine it with outdated GTK+ libraries on any Windows
+Of course you can decide to install older WeasyPrint versions with an
+older Python versions, combine it with outdated GTK+ libraries on any Windows
```
Comment (Tontyna) [inline docs/install.rst]: I liked `erstwhile`and `ancient`. In fact: I explicitly tried to avoid boring `older`.

File: docs/install.rst [hunk 3]
```diff
@@ -290,21 +290,20 @@ There's one thing you **must** observe:
 - If your Python is 32 bit you must use the 32 bit versions of those libraries.
 - If your Python is 64 bit you must use the 64 bit versions of those libraries.
 
-If you mismatch the bitness, the warning about kittens dying applies.
+If you install incompatible software architectures, the warning about kittens dying applies.
```
Comment (Tontyna) [inline docs/install.rst]: Not sure whether *software architecture* is more common than *bitness*

File: docs/install.rst [hunk 4]
```diff
@@ -290,21 +290,20 @@ There's one thing you **must** observe:
...
 
-In case you forgot which Python you installed, ask Python (in the console
-window):
+In case you forgot which Python architecture you installed, you can find out by runing the
```
Comment (Tontyna) [inline docs/install.rst]: running

File: docs/install.rst (hunk 3 above)
Comment (endalk200) [reply docs/install.rst]: architecture is more common in my opinion. OS settings and other books use it.

File: docs/install.rst (hunk 4 above)
Comment (endalk200) [reply docs/install.rst]: you're right my mistake.

**Comment by liZe** [comment]
Hello,

Thanks for your pull request! I agree with @Tontyna’s comments, so I’ll probably change a couple of things.

**Comment by Tontyna** [comment]
@liZe -- depending on cairocffi being a wheel or an egg, the instructions for Windows should mention the Visual C++ Tools

**Comment by liZe** [comment]
> @liZe -- depending on cairocffi being a wheel or an egg, the instructions for Windows should mention the Visual C++ Tools

Oh… As I don’t think that it’s possible to provide a wheel for cairocffi, we’ll have to change this in the documentation.

**Pull request merged**